src/github_utils.py

```diff
@@ -19,48 +19,56 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
-    print(response.text)
+    #print(response.text)
     return response.text
 
 
```

Comment:
`current_hunk` isn't needed since file is included in the hunk itself

```suggestion
Remove `current_file`
```